docs/source/lts_selection.org

```diff
@@ -0,0 +1,151 @@
+several versions of Iroha running in parallel, they might need to
+replicate the support work as many times as there are parallel
+versions.
+
+Instead of attempting to support multiple versions badly, we try to
+cater to 80% of use cases with 20% of the effort and reach a
+compromise between supporting every released version equally and only
+supporting the latest stable version.
+
+Specifically, we support the latest =stable= release. Often this
+release is either not as well-tested, well-documented, or
+well-supported by the SDKs, in addition to often changing in terms of
+its API and ABI, resulting in a moving target for compatibility. For
+this reason we designate a specific version that is at least one of 12
+releases to be the Long-term supported release following the criteria
+listed below.
+
+* LTS selection criteria
+
+The following is a non-exhaustive list of recommended considerations
+for selecting a particular version as an LTS candidate
+
+1. The release should be /unlikely/ to contain major /known/ issues which
+   would preclude it from being useful.
+
+   More specifically, the release should have a good coverage of a
+   basic use-case, for example, RC-6 was the last release which
+   contained the old consensus engine. Given that the actor-based
+   consensus had a maximum stable throughput of 10 TPS, RC-6 was not
+   labelled as an LTS, despite being used as one.
+
+   RC-9 contained all the necessary features for running an Iroha node
+   on the new consensus, some changes to the structure of the
+   configuration and a few fixes to the =p2p= subsystem. It could have
+   filled all the possible niches of any internal projects, and as
+   such it was labelled as an LTS release despite being less-well
+   tested.
+
+   Releases RC8, RC12, RC13 and RC17 were disqualified based on this
+   criterion.
+
+2. If there is a large feature that entails an API breakage, the
+   release that has the most stable version of the "old way of doing
+   things" should be prioritised as an LTS candidate.
+
+   For example, RC12 and RC13, were the last versions of Iroha that
+   still used a hard-coded permission validator. Them being labelled
+   as an LTS candidate was precluded by the major known issues,
+   particularly with stability, poor coverage, and because releasing
+   an LTS after RC9 would have split the development resources off
+   from fixing the issue of rigid un-modifiable permission validators.
+
+   RC14 and RC15 on the other hand, were in the process of a slow
+   migration of features. As such, the latter versions would be
+   considered direct upgrades over the previous one, having more
+   optimisations and better feature coverage.
+
+3. The release must be feature complete for a use-case.
+
+   This is different to point 1, because while RC10 and RC13 were free
+   from major problems, they did not offer sufficiently full coverage
+   of the features required for an internal project.
```

Comment:
Iroha is an open source project, we have few references to internal projects, I'm totally fine with that, though just for an idea, how about rephrasing that to something like 'Iroha's technical committee decision' (which is obviously represented by the most active contributors)
```suggestion
   of the features the technical committee has been guided the implementation by.
```